app/components/Profile.jsx

```diff
@@ -23,10 +27,31 @@ class Profile extends React.Component {
   });
  }
 
+ setClick(e) {
+  this.setState({e: this.props.profile.sets[e.target.value]})
+  this.forceUpdate()
+ }
+
+renderDeets(e) {
```

Comment:
I think we should name this something more normal in case somebody takes a look.